charts/agent-docker/templates/deployment.yaml

```diff
@@ -76,8 +76,14 @@ spec:
           - name: SCALR_TOKEN
             valueFrom:
               secretKeyRef:
+              {{- if .Values.agent.tokenFromSecret }}
```

Comment:
@huguesalary Could you please change the naming to tokenExistingSecret? We’re already using this in our second chart, and it would be better to keep the naming consistent: https://github.com/Scalr/agent-helm/blob/agent-k8s-0.5.40/charts/agent-k8s/templates/controller.yaml#L40-L45

Reply:
Sounds good 👌🏻